Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -1963,7 +1966,7 @@ String AccessibilityNodeObject::alternativeTextForWebArea() const
     }
 
     if (auto* owner = document->ownerElement()) {
-        if (owner->hasTagName(frameTag) || owner->hasTagName(iframeTag)) {
+        if (owner->elementName() == ElementName::HTML_frame || owner->elementName() == ElementName::HTML_iframe) {
```

Comment:
Can we get the elementName once? Would probably be shorter and avoid re-reading the same memory multiple times.